target/linux/omap/config-4.19

```diff
@@ -184,9 +197,11 @@ CONFIG_DEBUG_INFO=y
 CONFIG_DEBUG_LL_INCLUDE="mach/debug-macro.S"
 CONFIG_DEBUG_LOCK_ALLOC=y
 CONFIG_DEBUG_MUTEXES=y
+CONFIG_DEBUG_RT_MUTEXES=y
+CONFIG_DEBUG_RWSEMS=y
 CONFIG_DEBUG_SPINLOCK=y
```

Comment:
Do we need all these debug support?

Reply:
I don't think so, most of those are set because of CONFIG_PROVE_LOCKING from 68ff0d3d788eeb1880c997a8a9053bb6fdc37755. I'll add a patch to get rid of it

Reply:
Done, removed